Documentation/workflow/OfficalAndDailyBuilds.md

```diff
@@ -16,6 +16,12 @@ repositories.  You can see what is available from
  * CoreCLR at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>
  * NETCore.App at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App>
 
+These builds have a version number that follows the the versioning scheme described below (month number/day of month), 
+but they also will have a component that indicate which Git Branch the are working from 
+
+ * preview1 - are daily builds from the 'release/*' branch where * is the next official version to be released
+ * preview2 - are daily builds from the 'master' branch (where active work happens first (typically))
```

Comment:
This is going to be true like for next <2 weeks, and then these suffixes are going to change again.